server/repository/src/migrations/v1_08_00/assets/asset_log.rs

```diff
@@ -18,5 +18,14 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
         "#,
     )?;
 
+    if cfg!(feature = "postgres") {
+        sql!(
+            connection,
+            r#"
+                ALTER TYPE changelog_table_name ADD VALUE IF NOT EXISTS 'asset_log';
```

Comment:
I am on the fence here, mainly because I have an assumption that some migrations cannot be made with IF NOT EXISTS statement and that would make migrations fail ¯\_(ツ)_/¯ (since the column/trigger would already exist). I guess in this case you can just copy and paste the sql statement in, but once again I am not sure if it's actually testing that migration work in code. Would be great to have dev-ops tools/workflow to be able to save the state of the database pre-migration and then re-open that state at a particular branch.